NuGetResources/NuGetGalleryFeedMyGet.cs

```diff
@@ -0,0 +1,51 @@
+﻿//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+namespace Microsoft.PackageManagement.NuGetProvider
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Implements the gallery template for MyGet.
+    /// e.g. https://powershell.myget.org/feed/powershell-core/package/nuget/System.Management.Automation/6.0.0-beta.9
+    /// </summary>
+    public class NuGetGalleryFeedMyGet : INuGetGalleryFeed
```

Comment:
>NuGetGalleryFeedMyGet [](start = 17, length = 21)

Why do we need to add the separate implementation for myget feeds? If it is required, there are few other repostiories similar to myget, are we going to support others too?

Reply:
The gallery URL depends on the server implementation, hence the separate implementtions in NuGetProvider. As we discover their specific gallery feed URLs, yes, we will need to support all known NuGet gallery implementations.

---
In reply to: [154536885](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154536885) [](ancestors = 154536885)